sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClientBuilder.java

```diff
@@ -105,16 +103,6 @@
         ContainerRegistryAsyncClient.class
     })
 public final class ContainerRegistryClientBuilder {
-    private static final String CLIENT_NAME;
-    private static final String CLIENT_VERSION;
-
-    static {
-        Map<String, String> properties =
-            CoreUtils.getProperties("azure-containers-containerregistry.properties");
-        CLIENT_NAME = properties.getOrDefault("name", "UnknownName");
-        CLIENT_VERSION = properties.getOrDefault("version", "UnknownVersion");
-    }
```

Comment:
This was removed as it is no longer used and was moved into the Utils, correct?

Reply:
That is correct.